### ISSUE TITLE
Feature request: a command that takes a comment link to a text file dropped on discord...

### ISSUE THREAD

**Issue by calloatti** [FabricMC/discord-bot#4]
a command that takes a comment link to a text file dropped on discord as an argument.

* posts the text file to https://gist.github.com/ or similar.
* makes a comment in the same channel as the original comment, posting a link to the paste, the name of the original poster, link to the original file maybe?
* Deletes the original comment with said text file?

This would be to make it easier to read crash logs that people post without reading how to post crash logs.

Maybe make it so OP can delete the bot comment, or something like that, to prevent privacy issues.

**Comment by calloatti** [comment]
as an alternative make this automatic for files posted in some channels maybe?

**Comment by sfPlayer1** [comment]
Is this to mitigate the mobile discord client's ability to display attachments like desktop?

**Comment by calloatti** [comment]
Yes, but also the ability to display attachments on desktop is not that good tbh. Its hard to navigate, and scrolling horizontally is a pain, since the horizontal scrollbar is at the bottom, its almost invisible and not visible unless you scroll all the way down.

Net effect: I am a lot less inclined to help people that post a crash log file directly to discord instead of using a proper paste host, and I guess others may be acting the same way. (downloading 10+ message.txt files a day also gets old pretty fast)

And this happens sometimes:

![image](https://user-images.githubusercontent.com/12817862/119302439-80cd5f00-bc3a-11eb-8870-ca34f68741e6.png)

I guess in an ideal world txt/log files would just open in a web browser tab, no questions asked, and no leftover file in the downloads folder.
